bsp/src/org/jetbrains/bsp/project/importing/setup/FastpassConfigSetup.scala

```diff
@@ -55,11 +55,11 @@ class FastpassConfigSetup(processBuilder: ProcessBuilder) extends BspConfigSetup
       val stderrLines = stderrReader.lines()
       stderrLines.forEach{ line =>
         buildMessages = buildMessages.addError(line)
```

Comment:
use `.message` here too if you don't expect `line` to be actual errors in general